docs/military/darwin.md

```diff
@@ -4,4 +4,189 @@
+
+
+#### Auto-Release Wedge
+
+The Darwin auto-release wedges extend from the departure end runway threshold, 30 degrees splayed from the centreline, to 7nm  
+
+Auto-release at Darwin is only available for VFR aircraft and low-level IFR aircraft that conform with the TMP  
+
+Tower may depart aircraft that conform with the above conditions within the auto-release wedge using the following headings:
+
+|Departure Type| Runway 11| Runway 29|
+|---|---|----|
+|VFR |H070V, H106V or H140V| H260V, H286V or H320V|
+|IFR (Visual Dep)| H070V, H106V, or H140V |H260V, H286V or H320V|
+|IFR (Radar)| H070, H106 or H140 |H260, H286 or H320|
+
+!!! tip
+    Tower shall provide visual separation between aircraft in the auto-release wedge and subsequent departures until the preceding departure has been maneuvered by TCU
+
+#### Non-RNAV Departures
+For non-RNAV approved IFR aircraft with a MTOW of 25000kg or more, issue a Radar SID  
+For non-RNAV approved IFR aircraft with a MTOW of less than 25000kg, issue a visual departure or Radar SID  
+## Coordination
+### DN TCU / ENR
+#### Departures
+Voiceless coordination is in place from DN TCU to TRT for aircraft:  
+Planned at or above `F180`: `Assigned F180`  
+Planned below `F180`: `Assigned the RFL`  
+
+Any aircraft not meeting the above criteria must be prior coordinated to ENR.
+
+!!! example
+    <span class="hotline">**DN TCU** -> **TRT**</span>: "PFY1234, with your concurrence, will be assigned F160, for my separation with JTE654"  
+    <span class="hotline">**TRT** -> **DN TCU**</span>: "PFY1234, concur F160"  
+
+#### Arrivals
+The Standard assignable level from TRT to DN TCU is `A100`, and assigned a STAR. All other aircraft must be prior coordinated
+
+### DN TCU / DN ADC
+#### Auto Release
+
+Auto Release shall be used for aircraft that are:    
+a) Departing from a runway nominated on the ATIS; and  
+b) Issued a Procedural SID; and   
+c) Assigned `F180`.
+
+Any aircraft that don't meet these criteria must be coordinated to DN TCU with a "Next" Call.
+
+"Next" Coordination is a procedure where the **DN ADC** controller gives a heads-up to the DN TCU controller about an impending departure. The DN TCU controller will respond by assigning a heading to the aircraft, for the **DN ADC** controller to pass on with their takeoff clearance.
+
+
+!!! example
+    <span class="hotline">**DN ADC** -> **DN TCU**</span>: "Next, ABC, runway 11"  
+    <span class="hotline">**DN TCU** -> **DN ADC**</span>: "ABC, Track Extended Centreline, unrestricted"  
+    <span class="hotline">**DN ADC** -> **DN TCU**</span>: "Track Extended Centreline, unrestricted, ABC"  
+
+
+All other aircraft, including high level IFR and all departures from 18/36 require departure coordination with **DN TCU**
+
+### DN TCU Internal
+
+All aircraft transiting between internal DN TCU boundaries must be heads-up coordinated.
```

Comment:
This would be a bit of a pain (and no one would do it) . We have SID/STAR segregation, so can we have maybe Voiceless for
- Aircraft assigned the STAR and 1000ft above the ADC->TCU standard assignable
- Aircraft assigned the SID and F180

Reply:
Realworld ops. Maybe unwritten rule if both are on they can agree to waiving this requirement.